src/main/java/uk/gov/companieshouse/web/accounts/validation/smallfull/DirectorValidator.java

```diff
@@ -90,37 +90,40 @@
             validationErrors.add(error);
 
         } else {
-            assert addOrRemoveDirectors != null;
-            if (addOrRemoveDirectors.getExistingDirectors() == null ||
-                    Arrays.stream(addOrRemoveDirectors.getExistingDirectors())
-                            .noneMatch(d -> d.getResignationDate() == null)) {
-
-                ValidationError error = new ValidationError();
-                error.setFieldPath(DIRECTOR_TO_ADD);
-                error.setMessageKey(AT_LEAST_ONE_DIRECTOR_REQUIRED);
-                validationErrors.add(error);
+            if (addOrRemoveDirectors != null) {
+                if (addOrRemoveDirectors.getExistingDirectors() == null ||
+                        Arrays.stream(addOrRemoveDirectors.getExistingDirectors())
+                                .noneMatch(d -> d.getResignationDate() == null)) {
+
+                    ValidationError error = new ValidationError();
+                    error.setFieldPath(DIRECTOR_TO_ADD);
+                    error.setMessageKey(AT_LEAST_ONE_DIRECTOR_REQUIRED);
+                    validationErrors.add(error);
+                }
             }
         }
 
-        if(addOrRemoveDirectors.getExistingDirectors() != null) {
+        if (addOrRemoveDirectors != null) {
+            if (addOrRemoveDirectors.getExistingDirectors() != null) {
```

Comment:
Think you can combine these two if statements with &&

Reply:
As discussed, combined lines 106/107